kubevirt-vmware/build/Dockerfile

```diff
@@ -1,15 +0,0 @@
-FROM alpine:3.8
```

Comment:
should theses build files be deleted? It is hard to build the project without them :)

Reply:
This file was moved to new directory. I agree that it is needed to build the image.